extra/modules/confiant-ad-quality/src/main/java/org/prebid/server/hooks/modules/com/confiant/adquality/v1/ConfiantAdQualityBidResponsesScanHook.java

```diff
@@ -47,9 +56,14 @@ public Future<InvocationResult<AllProcessedBidResponsesPayload>> call(
             AuctionInvocationContext auctionInvocationContext) {
         final BidRequest bidRequest = getBidRequest(auctionInvocationContext);
         final List<BidderResponse> responses = allProcessedBidResponsesPayload.bidResponses();
+        final Map<Boolean, List<BidderResponse>> needScanMap = responses.stream()
+                .collect(Collectors.groupingBy(bidderResponse -> !biddersToExcludeFromScan.contains(bidderResponse.getBidder())));
+
+        final List<BidderResponse> toScan = Optional.ofNullable(needScanMap.get(true)).orElse(List.of());
+        final List<BidderResponse> avoidScan = Optional.ofNullable(needScanMap.get(false)).orElse(List.of());
```

Comment:
that looks better: `needScanMap.getOrDefault(true, Collections.emptyList())`

P.S. using `Collection.emptyList()` is preferable due to the style guide